filament/src/details/Engine.cpp

```diff
@@ -852,6 +852,12 @@ void FEngine::cleanupResourceListLocked(Lock& lock, ResourceList<T>&& list) {
 
 // -----------------------------------------------------------------------------------------------
 
+template<typename T>
+UTILS_ALWAYS_INLINE
+inline bool FEngine::isValid(const T* ptr, ResourceList<T>& list) {
+    return (ptr == nullptr) || (list.find(ptr) != list.end());
```

Comment:
a null pointer is a valid resource?

Reply:
> a null pointer is a valid resource?

good catch, I wrongly reused the code for `destroy` (where nullptr is considered ok, like `free()`)